changelogs/unreleased/gh-7486-prohibit-roll-back-due-to-replication_synchro_timeout.md

```diff
@@ -0,0 +1,16 @@
+## feature/replication
```

Comment:
Please change the version in both commits doc requests.
Should be `Since: 3.3`, because 3.2 is already released.